haas/test_common.py

```diff
@@ -59,6 +60,166 @@ def config_testsuite():
         })
 
 
+def initial_db():
```

Comment:
Maybe move this function into a separate file, or at the very least to the bottom of this file?

It's so big that it could cause the small helper functions to get lost.